content/right-sizing/reports-azure-right-sizing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Azure VM Right Sizing"
+#date: 2018-12-03
+draft: false
+categories:
+tags: ["#tools", "#azure", "#right sizing", "#getting started" ]
+author: Lawrence Lane
+---
+
+Compare your current virtual machines against Azure's most recent SKUs to find the best fit for your workloads and your budget.  The Azure VM Right Sizing report enables you to leverage your historic workloads to truly right-size your resources. You can add optional constraints (such as CPU utilization limits) to further contextualize recommendations and highlight different savings opportunities across your portfolio. By default, this report shows your top 10 recommendations.
+
+## Visualization Options
+
+Toggle between data visualizations by selecting the radio buttons **Cost vs CPU** or **None**.
+
+### Cost vs CPU
+
+![azure-cost-vs-cpu-visualization](/images/reports-azure-right-sizing/azure-cost-vs-cpu-visualization.png)
+
+### None (Table Only)
+
+![table-only-view](/images/reports-azure-right-sizing/table-only-view.png)
+
+## Table Options
+
+Explore your recommendations by drilling into specific VM details, or simply focus on the high-level SKU and pricing information found in the parent row.
+
+### Expand and Collapse Rows
+
+![vm-expand-rows](/images/reports-azure-right-sizing/vm-expand-rows.gif)
+
+### Sort Columns
+
+![vm-toggle-rows](/images/reports-azure-right-sizing/vm-rightsizing-toggle-rows.gif)
+
+---
+
+## Configuration
+
+You can configure the default report by defining a scope, setting constraints, and more.
+
+### 1. Scope of Analysis
+
+The following scope criteria can be used to both include or exclude resources:
+
+- **Names**: Select resources by inserting a string.
+- **Virtual Machines**: Select resources from a dropdown.
+- **Attributes**: Select resources based on attributes; supports `any` and `all` statements.
+- **Tags**: Select resource based on tags; supports `any` and `all` statements.
+
+### 2. CPU and Memory Constraints
```

Comment:
Two ideas for this suggestion:
1. Mention the ability to set a target utilization for both CPU and memory utilization.
2. Mention how memory utilization can be set manually (Default memory utilization when diagnostic monitoring not enabled) if guest OS diagnostic metrics haven't been enabled and link to https://docs.metricly.com/integrations/microsoft-azure/azure-enable-guest-os-diagnostic/.